test/controllers/users_controller_test.rb

```diff
@@ -167,6 +167,15 @@ class UsersControllerTest < ActionController::TestCase
     assert_nil user.jwt_secret
   end
 
+  test "User should be able to invalidate jwt for self" do
+    User.current = users(:one)
+    user = User.current
+    FactoryBot.build(:jwt_secret, token: 'test_jwt_secret', user: user)
```

Comment:
```suggestion
    FactoryBot.create(:jwt_secret, token: 'test_jwt_secret', user: user)
```
With `build` the object is not saved in the database, making the assertion pass even if the action was not performing correctly